src/js/components/Breadcrumbs.js

```diff
@@ -0,0 +1,242 @@
+import classNames from 'classnames';
```

Comment:
What is the difference between `Breadcrumbs` and `Breadcrumb` and how do they relate to the `Crumb` component?